media/js/mozorg/content-services-form.js

```diff
@@ -0,0 +1,122 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
+    'use strict';
+
+    var $sfForm = $('#sf-form');
+    var $sfFormSubmit = $('#sf-form-submit');
+
+    var $campaign_type = $('#campaign_type');
+    var $typeFields = $('.type-field');
+    var $typeInputs = $typeFields.find('input, textarea, select');
+
+    var $country = $('#country');
+    var $stateField = $('.state-field');
+    var $stateFlydown = $('#state');
+    var $provinceField = $('.province-field');
+    var $stateInput = $stateField.find('input, textarea, select');
+    var $provinceInput = $provinceField.find('input, textarea, select');
+
+    var $contentFormError = $('#content-form-error');
+    var $mainContent = $('#main-content');
+    var $htmlBody = $('html, body');
+
+    $sfForm.validate();
+
+    var scrollup = function() {
+        $htmlBody.animate({ scrollTop: $mainContent.offset().top }, 500);
+    };
+
+    // If Campagin Type is set to "Other", then show the description field
+
+    var campaign_type_other = function() {
+        return $campaign_type.val().indexOf('Other') > -1;
+        // TODO: Is this an l10n issue?
```

Comment:
This isn't a blocker for launch, but should be addressed before the page is localized.